### PR TITLE
Adds command to support getting editor a11y errors without UI

### PR DESCRIPTION
**Pull request by jcflow**

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -27,6 +28,10 @@ tinymce.create("tinymce.plugins.AccessibilityChecker", {
       )
     })
 
+    ed.addCommand("checkAccessibility", function (ui, {done, config, additional_rules}) {
+      nodeChecker.checkNode(ed.getBody(), done, config, additional_rules)
+    })
+
```
Comment (jcflow) [inline src/plugin.js:34]: The idea is getting the errors count in canvas-lms/packages/canvas-rce/src/RCEWrapper.js this way:

```
 editor.execCommand('checkAccessibility', {
      done: (errors) => {
        this.setState({ errors })
      }
    })
```

File: src/components/checker.js [hunk 2]
```diff
@@ -89,33 +89,18 @@ export default class Checker extends React.Component {
...
+      this.setState({ errorIndex: 0, errors, checking: false }, () => {
+        this.selectCurrent()
+        if (typeof done === "function") {
+          done()
         }
-      )
+      })
+      if (errors.length === 0) {
+        this._closeTimout = setTimeout(this.handleClose.bind(this), 3000)
+      }
```
Comment (gnbaron) [inline src/components/checker.js:101]: Why closing it automatically?

Comment (jcflow) [reply src/components/checker.js:101]: Hey! We are not closing automatically, the idea is set the **this** context in handleClose method using bind method. This fixes some tests issues while executing them.

Comment (gnbaron) [reply src/components/checker.js:101]: I've ran the tests without it and didn't get any failure. `bind` returns a function and this means `setTimeout` will run it later which apparently would close the component. Did you test it in canvas?

Comment (jcflow) [reply src/components/checker.js]: Tested it, and found a way to remove it

**Review by gnbaron — APPROVED** [review]

**Review by gnbaron — APPROVED** [review]

**Pull request merged**